docs/maps/maps-aggregations.asciidoc

```diff
@@ -40,6 +40,18 @@ This field will be used in the terms aggregation to group your documents into en
 This field will be used to sort your documents in the top hits aggregation.
 . Set *Documents per entity* to configure the maximum number of documents accumulated per entity.
 
+[role="xpack"]
+[[point-to-point]]
+=== Point to point
+
+Point to point plots aggregated data paths between the source and destination.
```

Comment:
The first part of the sentence is a little hard to parse.  How about:

A point-to-point connection plots aggregated data paths between the source and the destination.